python/lsst/pipe/supertask/cmdLineFwk.py

```diff
@@ -179,7 +179,7 @@ def parseAndRun(self, argv=None):
         else:
             # make execution plan (a.k.a. DAG) for pipeline
             graphBuilder = GraphBuilder(self.taskFactory, registry, args.data_query)
-            qgraph = graphBuilder.makeGraph(pipeline)
+            qgraph = graphBuilder.makeGraph(pipeline, collection)
```

Comment:
I'm thinking of it as "can we invoke `makeGraph()` more than once on the same graph builder instance"? In that sense taskFactory and registry may be a reasonable state you can keep between those calls but data_query should probably be moved into makegraph() arguments. OTOH I can see that it can be a single-shot instance too. I guess I'd like to see more use cases to decide on what interface should look like.